lib/makeup/lexers/erlang_lexer.ex

```diff
@@ -178,6 +178,14 @@ defmodule Makeup.Lexers.ErlangLexer do
   syntax_operators =
     word_from_list(~W[+ - +? ++ = == -- * / < > /= =:= =/= =< >= ==? <- ! ? ?!], :operator)
 
+  record =
+    token(string("#"), :operator)
+    |> concat(atom)
```

Comment:
I thought about doing something like `|> concat(token(atom_name, :operator))` as to give it the same color as the `#` in the beginning, wdyt?

<details>
<summary>Screenshot</summary>
<img width="424" alt="Screenshot 2019-10-15 11 37 35" src="https://user-images.githubusercontent.com/14015177/66844154-3951fd80-ef44-11e9-9a03-b4fe437aaf65.png">

</details>

Reply:
I like the current approach more. `#` is not really part of the record name.